src/calculateRentalCost.js

```diff
@@ -4,7 +4,18 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRentalCar = 40;
+  let discount = 0;
+
+  if (days >= 3 && days <= 6) {
+    discount = 20;
+  } else if (days >= 7) {
+    discount = 50;
+  }
```

Comment:
You can simplify the conditional logic by using 'if' with 'return' and removing the 'else' block, as suggested in the checklist: 'Prefer if with return over if else to simplify later conditions.'